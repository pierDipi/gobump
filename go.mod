module github.com/pierdipi/gobump

go 1.14

require golang.org/x/mod v0.3.0
