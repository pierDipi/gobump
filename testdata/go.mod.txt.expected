module github.com/pierdipi/gobump

go 1.15

require golang.org/x/mod v0.3.0
